dev-manuals/architecture/memory/README.md

```diff
@@ -0,0 +1,45 @@
+# Architecture of Memory
+
+## Purpose
+
+```md
```

Comment:
Why do you put normal text into a code block?
I mean it doesn't look bad but I think we don't do that on other Readmes

Reply:
UML class diagrams for vue applications are not that easy to do because it uses a lot of HTML. Therefore, I googled it and found the approach to use "Uses" arrows.
For the readme I copied the template which inluded this formating. I didn't change it.